wiki/History_of_osu!/Online_rankings/osu!/de.md

```diff
@@ -0,0 +1,390 @@
+| ::{ flag=FI }:: [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+| ::{ flag=DE }:: [WhiteCat](https://osu.ppy.sh/users/4504101) | 1 |
+| ::{ flag=JP }:: [zaru-d](https://osu.ppy.sh/users/79678) | 1 |
+
+## Längste zusammengezählte Titelverteidigung
+
+*Anmerkung: Der aktuelle Spieler auf dem ersten Platz ist nicht in dieser Liste enthalten.*
+
+Die folgende Liste enthält die Gesamtanzahl der Tage, die ein Spieler den Titel des ersten Platzes getragen hat (Stand: 3. Juni 2021).
+
+| Spieler | Länge in Tagen |
+| --: | :-: |
+| ::{ flag=KR }:: [chocomint](https://osu.ppy.sh/users/124493) | 967 |
+| ::{ flag=TW }:: [hvick225](https://osu.ppy.sh/users/50265) | 578 |
+| ::{ flag=DE }:: [WhiteCat](https://osu.ppy.sh/users/4504101) | 550 |
+| ::{ flag=PL }:: [WubWoofWolf](https://osu.ppy.sh/users/39828) | 507 |
+| ::{ flag=US }:: [Saturos](https://osu.ppy.sh/users/3781) | 272 |
+| ::{ flag=UY }:: [H1ko](https://osu.ppy.sh/users/58710) | 266 |
+| ::{ flag=PL }:: [Rafis](https://osu.ppy.sh/users/2558286) | 238 |
+| ::{ flag=CL }:: [Mathi](https://osu.ppy.sh/users/5339515) | 213 |
+| ::{ flag=US }:: [Cyclone](https://osu.ppy.sh/users/18589) | 196 |
+| ::{ flag=GB }:: [Doomsday](https://osu.ppy.sh/users/18983) | 172 |
+| ::{ flag=JP }:: [rrtyui](https://osu.ppy.sh/users/352328) | 144 |
+| ::{ flag=US }:: [idke](https://osu.ppy.sh/users/4650315) | 137 |
+| ::{ flag=US }:: [Vaxei](https://osu.ppy.sh/users/4787150) | 77 |
+| ::{ flag=KR }:: [FlyingTuna](https://osu.ppy.sh/users/9224078) | 59 |
+| ::{ flag=JP }:: [zaru-d](https://osu.ppy.sh/users/79678) | 59 |
+| ::{ flag=KR }:: [sayonara-bye](https://osu.ppy.sh/users/713266) | 58 |
+| ::{ flag=PL }:: [Kubu](https://osu.ppy.sh/users/29130) | 51 |
+| ::{ flag=CN }:: [Ana_Coppola](https://osu.ppy.sh/users/88717) | 44 |
+| ::{ flag=US }:: [Ivalset](https://osu.ppy.sh/users/827) | 39 |
+| ::{ flag=TW }:: [Uan](https://osu.ppy.sh/users/147623) | 37 |
+| ::{ flag=KR }:: [firebat92](https://osu.ppy.sh/users/1777162) | 35 |
+| ::{ flag=KR }:: [Reimu-Desu](https://osu.ppy.sh/users/948713) | 35 |
+| ::{ flag=AU }:: [eyup](https://osu.ppy.sh/users/88) | 33 |
+| ::{ flag=US }:: [geckogates](https://osu.ppy.sh/users/252524) | 33 |
+| ::{ flag=US }:: [Rolled](https://osu.ppy.sh/users/5243) | 27 |
+| ::{ flag=KR }:: [Sure](https://osu.ppy.sh/users/70730) | 26 |
+| ::{ flag=DE }:: [ShadowSoul](https://osu.ppy.sh/users/494970) | 20 |
+| ::{ flag=US }:: [Freddie Benson](https://osu.ppy.sh/users/7342622) | 13 |
+| ::{ flag=JP }:: [tobebuta](https://osu.ppy.sh/users/14053) | 13 |
+| ::{ flag=JP }:: [Flute](https://osu.ppy.sh/users/211278) | 8 |
+| ::{ flag=KR }:: [La Valse](https://osu.ppy.sh/users/70863) | 7 |
+| ::{ flag=NZ }:: [Echo](https://osu.ppy.sh/users/431) | 6 |
+| ::{ flag=TW }:: [f0012979](https://osu.ppy.sh/users/12143) | 4 |
+| ::{ flag=CN }:: [5231_Kinoko](https://osu.ppy.sh/users/181057) | 1 |
+| ::{ flag=FI }:: [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+
+## Längste ununterbrochene Titelverteidigung
```

Comment:
A few alternatives that came to my mind:

- `durchgehend`
- `fortlaufend`
- `kontinuierlich`
- `zusammenhängend`

Though I prefer the current version. It emphasises that there was no *break* in-between.

Reply:
Current one is good yeah